apps/jupyter/extensions/notebook-runner/src/iframe-api.ts

```diff
@@ -121,6 +127,13 @@ export class EmbeddedPythonCallbacks {
         e,
       );
 
+      const errorMessage = e instanceof Error ? e.message : String(e);
+      toast.error(
+        getMessage(this.translator, MessageKeys.CannotGetTask, {
+          error: errorMessage,
+        }),
+      );
```

Comment:
I would suggest two things here:

Create a custom class `XYZ` wrapping `ITranslator` and exposing the `getMessage` method.
Then all you need to do, is pass around this class and not `ITranslator`.

Moreover, it would allow you do add a method specific to handle errors since you have the same error to string logic in multiple places now.

Reply:
True, sorry my comment was wrong here. Funny because I saw the AppTranslator yesterday but this morning I saw this when searching for `getMessage`

<img width="1334" height="1440" alt="image" src="https://github.com/user-attachments/assets/f8c6025f-ad69-495c-b53f-0b4adc06241e" />


and missed the `AppTranslator` below. I was confused but should have question more, sorry!

In that case I would just remove these additional exported methods, I guess there is no need for them anymore now that there is the `AppTranslator` class, right?